articles/active-directory/develop/active-directory-certificate-credentials.md

```diff
@@ -21,7 +21,7 @@ ms.custom: aaddev
 
 # Certificate credentials for application authentication
 
-Azure Active Directory allows an application to use its own credentials for authentication, for example, in the OAuth 2.0 Client Credentials Grant flow and the On-Behalf-Of flow.
+Azure Active Directory allows an application to use its own credentials for authentication, for example, in the OAuth 2.0 [Client Credentials Grant flow](https://docs.microsoft.com/en-us/azure/active-directory/develop/active-directory-protocols-oauth-service-to-service) [(v2)](https://docs.microsoft.com/en-us/azure/active-directory/develop/active-directory-v2-protocols-oauth-client-creds#second-case-access-token-request-with-a-certificate) and the [On-Behalf-Of flow](https://docs.microsoft.com/en-us/azure/active-directory/develop/active-directory-protocols-oauth-on-behalf-of) [(v2)](https://docs.microsoft.com/en-us/azure/active-directory/develop/active-directory-v2-protocols-oauth-on-behalf-of).
 One form of credential that can be used is a JSON Web Token(JWT) assertion signed with a certificate that the application owns.
```

Comment:
This is better but still confusing as to where is the v1 link. May I suggest that we do it this way:
Client Credentials Grant flow (v1, v2) and On-Behalf-Of flow (v1, v2) , where v1 and v2 link to the respective articles.

Reply:
Nice, I like that better 🔥

Reply:
Hi @navyasric - please see the update, thanks.